app/javascript/packs/base.jsx

```diff
@@ -144,3 +144,25 @@ getInstantClick().then((spa) => {
 });
 
 initializeNav();
+
+async function loadCreatorSettings() {
+  try {
+    const [{ CreatorSettingsController }, { Application }] = await Promise.all([
+      import('@controllers/creator_settings_controller'),
+      import('@hotwired/stimulus'),
+    ]);
+
+    const application = Application.start();
+    application.register('creator-settings', CreatorSettingsController);
+  } catch (error) {
+    Honeybadger.notify(
+      `Error loading the creator settings controller: ${error.message}`,
+    );
+  } finally {
+    // TODO: Friendly error message if the controller fails to load
+  }
+}
+
+if (document.location.href.endsWith('/admin/creator_settings/new')) {
```

Comment:
TODO: I need to look @citizen428's PR that added the routes gem for frontend code instead of using a hard-coded value here.

Reply:
Probably just need to use: `window.Forem.routes.new_admin_creator_setting_path()`

Reply:
I think it's something like this

```
import { newAdminCreatorSettingPath } from 'routes.js.erb';
console.log(newAdminCreatorSettingPath());
```

Reply:
(blocking): Also, when a user directs from the login screen the url will be `http://localhost:3000/admin/creator_settings/new?referrer=http://localhost:3000/` hence the condition `document.location.href.endsWith('/admin/creator_settings/new')` will evaluate to `false`.